dataflow/src/node/debug.rs

```diff
@@ -40,33 +50,42 @@ impl Node {
             },
             None => format!("{} / -", idx.index()),
         };
+
         match self.inner {
             NodeType::Source => s.push_str("(source)"),
             NodeType::Dropped => s.push_str("✗"),
-            NodeType::Ingress => s.push_str(&format!("{{ {} | (ingress) }}", addr)),
-            NodeType::Egress { .. } => s.push_str(&format!("{{ {} | (egress) }}", addr)),
+            NodeType::Ingress => s.push_str(&format!(
+                "{{ {{ {} {} }} | (ingress) }}",
+                addr,
+                materialized
+            )),
+            NodeType::Egress { .. } => {
```

Comment:
I don't think an `Egress` node would currently ever be materialized (@jonhoo -- ?), though this code is obviously still correct in that case.

Reply:
Yeah, `Egress` is never materialized.